### PR TITLE
calculate dropped connections directly in the agent

### PR DESCRIPTION
**Pull request by kky**
To avoid any irregularities in roll-ups / averages, etc. Also fix a
typo (“conn”=>”handled”)

### CODE REVIEW

File: checks.d/nginx.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def parse_text(cls, raw, tags):
             handled = int(parsed.group(2))
             requests = int(parsed.group(3))
             output.extend([('nginx.net.conn_opened_per_s', conn, tags, 'rate'),
-                           ('nginx.net.conn_handled_per_s', conn, tags, 'rate'),
+                           ('nginx.net.conn_dropped_per_s', conn-handled, tags, 'rate'),
```
Comment (remh) [inline checks.d/nginx.py]: Thanks a lot for this @kky !

Tiny nitpick could you add a whitespace on each side of the - sign to respect PEP8 code style please ?

Soit would be:
```python
('nginx.net.conn_dropped_per_s', conn - handled, tags, 'rate'),
```

Thanks again!

**Comment by alq666** [comment]
Will it affect existing dashboards and alerts or is this a new metric altogether?

**Comment by kky** [comment]
@remh done
@alq666 this is a new metric altogether; conn_handled_per_s never made it into the wild.

**Comment by remh** [comment]
Looks good to me. Thanks!

**Pull request merged**